src/test/java/bio/terra/janitor/service/janitor/JanitorServiceTest.java

```diff
@@ -87,4 +90,67 @@ public void createResource_Duplicates() {
     assertThat(retrievedStates, Matchers.hasEntry(thirdId, "READY"));
     assertThat(retrievedStates, Matchers.aMapWithSize(3));
   }
+
+  @Test
+  public void abandonThenBumpResources() throws Exception {
+    CloudResourceUid resourceUid =
+        new CloudResourceUid()
+            .googleProjectUid(new GoogleProjectUid().projectId(UUID.randomUUID().toString()));
+    String firstId =
+        janitorService
+            .createResource(
+                new CreateResourceRequestBody()
+                    .resourceUid(resourceUid)
+                    .creation(DEFAULT_TIME)
+                    .expiration(DEFAULT_TIME),
+                ADMIN_USER)
+            .getId();
+
+    String secondId =
```

Comment:
I think we have a second id to test that a duplicated resource is not bumped, which is important business logic so that we don't have duplicate ready resources. add a comment somewhere.